metapool/sample_sheet.py

```diff
@@ -571,6 +640,10 @@ def _add_metadata_to_sheet(self, metadata, sequencer):
 
         # Note: 'iseq' should remain at the tail of this list, since it
         # is a substring of the others.
+        # TODO: It makes me super nervous to have this list here and to also
```

Comment:
Excellent point and I agree on moving them to the same are of the code. Could you open an issue or address?